### PR TITLE
Hotfix/faq questions

### PR DESCRIPTION
**Pull request by kumardennis**

### CODE REVIEW

**Comment by edanchenkov** [comment]
@kumardennis there is a check failing. Also why is it called `faq2` and not `faq` ?

**Comment by kumardennis** [comment]
> @kumardennis there is a check failing. Also why is it called `faq2` and not `faq` ?

@edanchenkov  it says ```missing script: check``` , should i installl it is it gonna be fixed after merge, coz last time i wasnt supposed to install. 

The folder was called faq2 , with faq it just wasnt reading and showing the faq page, idont know why.

**Comment by edanchenkov** [comment]
@kumardennis you should merge from `develop`, I believe we have already talked about it before.
Well, you need to figure out why it does not work for `faq`. I recommend to clean gatsby files by running `npm run clean` and try again.

File: src/pages/faq/q13.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+path: "/faq2"
```
Comment (edanchenkov) [inline src/pages/faq/q13.mdx]: @kumardennis could you update this?

**Pull request merged**